ports/psoc6/mtb-libs/Makefile

```diff
@@ -42,7 +42,7 @@ MTB_TYPE=COMBINED
 # configurations. If TARGET is manually edited, ensure TARGET_<BSP>.mtb with a
 # valid URL exists in the application, run 'make getlibs' to fetch BSP contents
 # and update or regenerate launch configurations for your IDE.
-TARGET=APP_CY8CPROTO-062-4343W
+TARGET=APP_CY8CPROTO-063-BLE
```

Comment:
Don´t change this so far, as for the ci multi-board-support this is not yet set active when programming etc. So it will break all on-target-test of the previous board.
Soon, it won´t matter much this value :)

Reply:
Yep! I last tested it with new board and the changes remained I guess while pushing. I will revert.